src/components/App/App.tsx

```diff
@@ -93,6 +94,9 @@ function App() {
   return (
     <main>
       <Switch>
+        <Route exact path="/">
+          <LandingPage />
```

Comment:
It looks like this is routed and supposed to render from the code but i haven't seen the component on my local..